tailwind.config.js

```diff
@@ -3,20 +3,24 @@ module.exports = {
   content: ["./src/**/*.{js,jsx,ts,tsx}"],
   theme: {
     extend: {
+      backgroundImage: {
+        "galaxy-pattern": "url('assets/galaxy.png')",
+      },
       fontFamily: {
-        sans: ["OpenSans", "sans-serif"],
-        audioWide: ["AudioWide"]
+        sans: ["IBMPlexSans", "OpenSans", "sans-serif"],
+        audioWide: ["AudioWide"],
+        ibmPlex: ["IBMPlexSans"],
```

Comment:
If this is already your default sans font, why have another?

Reply:
Yea the font changed so right now we don't need AudioWide or OpenSans, forgot to remove them